enterprise/utils.py

```diff
@@ -663,3 +666,31 @@ def track_event(user_id, event_name, properties):
                 'clientId': tracking_context.get('client_id')
             }
         })
+
+
+def parse_datetime_generic(datetime_value):
```

Comment:
@UmanShahzad Looks like the main purpose of this function is to silence errors that `parse_datetime` might throw. Maybe we can change the name to better reflect that? How about something like `parse_datetime_handle_invalid` or `parse_datetime_allow_invalid`?

Reply:
I'm going to try, but I think pylint might shut me down. The proposed function names have 2 verbs in them (parse/handle, parse/allow), so that's why I chose something like "_generic" as a prefix, in an attempt to imply that it can handle any type. Maybe `_generic_type` would have been a better prefix. Let's see.

Reply:
Ok, `_handle_invalid` worked as a suffix here, and pylint didn't complain :+1: